Breadcrumbs/Breadcrumbs.vue

```diff
@@ -14,43 +32,187 @@ const breadcrumbs = computed(() => {
     },
     ...breadcrumbsInjected.value,
   ]
-    .flatMap(breadcrumb => [breadcrumb, 'splitter'])
-    .slice(0, -1)
+})
+
+const breadcrumbEls = ref<(Element | ComponentPublicInstance | null)[]>([])
+
+const breadcrumbsDivEl = ref<HTMLElement>()
+const { width: breadcrumbsDivWidth } = useElementSize(breadcrumbsDivEl)
+
+const slotAppendEl = ref<HTMLElement>()
```

Comment:
You are missing the `right` slot. Also, can we not just check the wrapper's width?

Reply:
I think the right slot isn't necessary, because my calculation are based on the breadcrumbs div el, which is smaller the bigger #right slot is.